universal-application-tool-0.0.1/app/views/style/ApplicantStyles.java

```diff
@@ -106,7 +106,7 @@
   public static final String BUTTON_REVIEW =
       StyleUtils.joinStyles(BUTTON_BASE_OUTLINE_UPPERCASE, Styles.TEXT_BASE);
   public static final String BUTTON_SUBMIT_APPLICATION =
-      StyleUtils.joinStyles(BUTTON_BASE_SOLID_UPPERCASE, Styles.TEXT_BASE, Styles.MX_AUTO);
+      StyleUtils.joinStyles(BUTTON_BASE_SOLID_UPPERCASE, Styles.TEXT_BASE, Styles.MX_AUTO,Styles.BG_GREEN_50);
```

Comment:
(blocking) this is close, but BG_GREEN_50 is way too light to see. A screenshot of the page would be really helpful.